### PR TITLE
[pre-commit] Ensure a file is either empty, or ends with one newline

### PR DESCRIPTION
**Pull request by easyCZ**
## Description

See https://pre-commit.com/hooks.html

## How to test

Build runs

## Release Notes

```release-note
NONE
```

## Documentation

NONE

### CODE REVIEW

**Review by andrew-farries — APPROVED** [review]

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ repos:
     hooks:
       #- id: check-yaml
       #  args: [--allow-multiple-documents]
-      # - id: end-of-file-fixer
+      - id: end-of-file-fixer
```
Comment (csweichel) [inline .pre-commit-config.yaml:7]: Why was this disabled in the first place?

Comment (csweichel) [reply .pre-commit-config.yaml:7]: @aledbf do you know the reason? (context: https://github.com/gitpod-io/gitpod/commit/f65deb95b14fdf6658b14445b3252026120818c9)

Comment (aledbf) [reply .pre-commit-config.yaml:7]: It was related to the number of changes at the time of introduction of `pre-commit`

**Comment by mustard-mh** [comment]
/hold

/werft run with-clean-slate-deployment=true

This PR block merge pool

**Pull request merged**